packages/lesswrong/components/posts/PostsEditBotTips.tsx

```diff
@@ -0,0 +1,139 @@
+import React from "react";
+import { registerComponent, Components } from "../../lib/vulcan-lib";
+import { isEAForum } from "../../lib/instanceSettings";
+import { Link } from "../../lib/reactRouterWrapper";
+import classNames from "classnames";
+import { useTracking } from "../../lib/analyticsEvents";
+
+const styles = (theme: ThemeType) => ({
+  root: {
+    position: 'absolute',
+    right: -390,
+    top: -100,
+    height: '120%',
+    '@media (max-width: 1500px)': {
+      right: -335,
+    },
+    '@media (max-width: 1360px)': {
+      right: -268,
+    },
+    '@media (max-width: 1234px)': {
+      position: 'static',
+      marginTop: 16
```

Comment:
Hmm I can see why you did it with absolute positioning but this does seem fairly brittle. Did you try box at a higher level in the layout (like in PostsNewForm and PostsEditForm)?

Reply:
Yeah that's actually how I originally designed it (inserting it one level up, positioning it with a grid on those components) but I moved it here mainly to make it easier to correctly position the card on both desktop (where it's on the right side) and mobile (where it's under the editor). There were other benefits too: now there's less code repeated, and all the logic is more centralized, and I don't need to wrap the parent components in a grid. The grid wasn't that useful anyway since I didn't want the card to change widths much.